package.json

```diff
@@ -4,35 +4,40 @@
   "private": true,
   "dependencies": {
     "@privacybydesign/irma-frontend": "^0.4.3",
-    "i18next": "^21.3.3",
+    "i18next": "^22.4.9",
     "moment": "^2.29.1",
     "prop-types": "^15.7.2",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-i18next": "^11.12.0",
-    "react-redux": "^7.2.5",
+    "react": "^18.2.0",
```

Comment:
I saw this warning in the console of my browser:
```
Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot
```

Reply:
As did I, let me check: If it's a relative small change I can take it into account, otherwise it would be better if we'd make it a separate refactoring task.

Reply:
Done